Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ubuntu:20.04
+FROM ubuntu:22.04 AS build
```

Comment:
Could you add `git` to `RUN apt install -y wget gnupg2`? This will allow us to add the proper version to the final binary.

Reply:
I've added `git` to the `apt install`.